inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleReadableMetaData.java

```diff
@@ -447,16 +448,22 @@ public static Map<String, String> getOracleType(@Nullable TableChanges.TableChan
         }
         Map<String, String> oracleType = new LinkedHashMap<>();
         final Table table = tableSchema.getTable();
-        table.columns()
-                .forEach(
-                        column -> {
-                            oracleType.put(
-                                    column.name(),
-                                    String.format(
-                                            "%s(%d)",
-                                            column.typeName(),
-                                            column.length()));
-                        });
+        for (Column column : table.columns()) {
+            if (column.typeName().matches("\\w.+\\([\\d ,]+\\)")) {
```

Comment:
It is recomment to give a reason for the handle?

Reply:
done.